carts-coherence/src/main/java/io/helidon/examples/sockshop/carts/coherence/CoherenceCartRepository.java

```diff
@@ -0,0 +1,91 @@
+/*
+ *  Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.helidon.examples.sockshop.carts.coherence;
+
+import io.helidon.examples.sockshop.carts.Cart;
+import io.helidon.examples.sockshop.carts.DefaultCartRepository;
+
+import io.helidon.examples.sockshop.carts.Item;
+import javax.annotation.Priority;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Alternative;
+
+import javax.inject.Inject;
+
+import com.oracle.coherence.cdi.Cache;
+import com.tangosol.net.NamedCache;
+import org.eclipse.microprofile.opentracing.Traced;
```

Comment:
organize imports by grouping them into `io.helidon`, `javax`, `com.oracle/tangosol` and `org.eclipse` sections